### DISCUSSION TITLE
Suggested Preset for Extreme Speed

### DISCUSSION THREAD

**Discussion by MohammadAboulEla** [lllyasviel/Fooocus#2621] [Ideas]
I have noticed that Extreme Speed preset does not give any good result at all, so I suggest using "juggernautXL_v9Rdphoto2Lightning.safetensors" model with 6 step and cfg of 2 as an alternative, it is even faster than the current Extreme Speed Preset and the result are way better.
![2024-03-23_21-34-24_4864](https://github.com/lllyasviel/Fooocus/assets/89942942/2e815f44-33f3-4912-bdc6-3334a9df9d0b)

**Comment by mashb1t** [comment]
We keep Extreme Speed (LCM) for now, also Lightning has just been introduced, which works with every model (except already lightning ones with baked-in lightning LoRA).
Feel free to create a new preset for yourself with whatever settings you desire!

**Comment by PeterJared** [comment]
> I have noticed that Extreme Speed preset does not give any good result at all, so I suggest using "juggernautXL_v9Rdphoto2Lightning.safetensors" model with 6 step and cfg of 2 as an alternative, it is even faster than the current Extreme Speed Preset and the result are way better. ![2024-03-23_21-34-24_4864](https://private-user-images.githubusercontent.com/89942942/316276511-2e815f44-33f3-4912-bdc6-3334a9df9d0b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEyNzcwNzUsIm5iZiI6MTcxMTI3Njc3NSwicGF0aCI6Ii84OTk0Mjk0Mi8zMTYyNzY1MTEtMmU4MTVmNDQtMzNmMy00OTEyLWJkYzYtMzMzNGE5ZGY5ZDBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI0VDEwMzkzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExN2ViZGIzNGUxYTEyYjNlYTRhZDE0M2I0YWVjNTEyZDUwZmQwMjE3MGI5NGUyMWRiOTk3NjA4ZDUzN2M4M2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ES4cRjTkNZJKP1jMDYAQTYZyp3ZgPNT2X8OqwceRTFc)

Thanks for the tip, I also think lighting default settings are not what I needed and I was looking for for alternatives. I will test your version. I quite happy with turbo (curiously no default presets) because the quality is similar to no turbo but faster.

**Reply by mashb1t** [reply]
fyi https://github.com/lllyasviel/Fooocus/pull/1580
